src/App.css

```diff
@@ -21,6 +21,10 @@
   overflow: hidden;
 }
 
+.undo {
```

Comment:
We use CSS grid, so the grid template should be changed instead of adding margin.